File: 13.c
#include <stdio.h>

int main() 
{
    int a, b, c, x;
    printf("Enter first number: ");
    scanf("%d", &a);
    printf("Enter second number: ");
    scanf("%d", &b);
    printf("Enter third number: ");
    scanf("%d", &c);
    if (a <= b && a <= c) 
    {
        x = a;
    }
    else if (b <= a && b <= c) 
    {
        x = b;
    }
    else
    {
        x = c;
    }
    printf("Smallest number is: %d\n", x);
    return 0;
}

File: 13.js
const readline = require("readline");
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
});
let numbers = [];
rl.question("Enter first number: ", function (a) {
    numbers.push(parseFloat(a));
    rl.question("Enter second number: ", function (b) {
        numbers.push(parseFloat(b));
        rl.question("Enter third number: ", function (c) {
            numbers.push(parseFloat(c));
            const x = Math.min(...numbers);
            console.log("Smallest number is: " + x);
            rl.close();
        });
    });
});
